### PR TITLE
ug: el2g: add warning about deprecation

### PR DESCRIPTION
**Pull request by vanmaegima**
This is not supported since Oct 1st 2024.

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 042c2a149df334ed115b38dee2c250f3a22333e6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2773/docs/artifacts/html/index.html

**Review by doanac — APPROVED** [review]
this is fine. i wouldn't even mind just deleting this - the next LmP could be months from now and we already don't support it.

**Comment by mike-scott** [comment]
> this is fine. i wouldn't even mind just deleting this - the next LmP could be months from now and we already don't support it.

My concern with deleting immediately is that folks won't know we stopped supporting this.

File: source/user-guide/el2g.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Integrating NXP EdgeLock 2GO
 ============================
 
+.. warning::
+
+   The content in this page is not supported in current LmP. This page will be deleted in the next LmP release (v96).
```
Comment (kprosise) [inline source/user-guide/el2g.rst:8]:
```suggestion
   The content on this page is not supported by the current LmP. This page will be deleted with the next release (v96).
```

**Review by kprosise — APPROVED** [review]
minor little tweaks suggested, but it LGTM.

**Comment by kprosise** [comment]
@mike-scott @doanac @vanmaegima I think keeping this page with the added warning until v96 is wise. As mike-scott mentioned, it lets folks know it is no longer supported. I worry about someone googling it, and ending up being directed to a outdated version...not that this will prevent that, but perhaps will help reduce that from happening?

Additionally, *when* we do delete this—or any feature page—we may want to add a redirect...perhaps to a generic, non-indexed page stating it was deprecated, and a link back to the home page? I can do this when it comes time, if we think it would be a good idea.

**Comment by vanmaegima** [comment]
@kprosise I've addressed your comments

**Comment by doanac** [comment]
Docs for b41dbbc0b6705d9ce2c94784cc549b2ddde034fe are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2774/docs/artifacts/html/index.html

**Pull request merged**